rule-extractor/src/main/java/org/sonarsource/sonarlint/core/rule/extractor/SonarLintRuleDefinition.java

```diff
@@ -62,6 +67,15 @@ public SonarLintRuleDefinition(RulesDefinition.Rule rule) {
     this.name = rule.name();
     this.defaultSeverity = IssueSeverity.valueOf(rule.severity());
     this.type = RuleType.valueOf(rule.type().name());
+    var cleanCodeAttributeFromRule = rule.cleanCodeAttribute();
+    this.cleanCodeAttribute = cleanCodeAttributeFromRule == null ?
+      // TODO Should we keep this as a transition feature? When do we drop this default value?
```

Comment:
What is the goal of this default value? On the client side, I would expect a null value in the case of old CCT

Reply:
IIRC this was to get non-null values returned while we wait for plugins to be updated. Probably we can remove this now.

Reply:
Hi @jblievremont, can you remove the default values maybe already tomorrow? Would help me (at least) a lot with testing ^^

Reply:
Merci 🙇‍♂️